docs/termux.md

```diff
@@ -0,0 +1,61 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+# Termux
+
+- [Termux](#termux)
+  - [Installation](#installation)
+  - [Upgrading](#upgrading)
+  - [Known Issues](#known-issues)
+    - [Search issue](#search-issue)
+    - [Backspace not working](#backspace-not-working)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Termux
+
+Termux is an Android terminal application and Linux environment, which can also run code-server from your phone.
+
+## Installation
+
+1. Install Termux from the [Google Play Store](https://play.google.com/store/apps/details?id=com.termux&hl=en_US&gl=US)
```

Comment:
suggest we omit the language settings here, otherwise that seems kinda rude for people that prefer/have configured different languages :stuck_out_tongue: 

```suggestion
1. Install Termux from the [Google Play Store](https://play.google.com/store/apps/details?id=com.termux)
```

Reply:
Didn't even notice it! Fantastic catch 🚀